jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/MetaInfConfiguration.java

```diff
@@ -720,25 +718,21 @@ public FileVisitResult visitFile(Path f, BasicFileAttributes attrs) throws IOExc
      */
     public Collection<URL> getTlds(URI uri) throws IOException
     {
-        HashSet<URL> tlds = new HashSet<URL>();
-
+        HashSet<URL> tlds = new HashSet<>();
         URI jarUri = uriJarPrefix(uri, "!/");
-        URL url = jarUri.toURL();
-        JarURLConnection jarConn = (JarURLConnection)url.openConnection();
-        jarConn.setUseCaches(Resource.getDefaultUseCaches());
-        JarFile jarFile = jarConn.getJarFile();
-        Enumeration<JarEntry> entries = jarFile.entries();
-        while (entries.hasMoreElements())
-        {
-            JarEntry e = entries.nextElement();
-            String name = e.getName();
-            if (name.startsWith("META-INF") && name.endsWith(".tld"))
+        try (Resource.Mount mount = Resource.mount(jarUri);
```

Comment:
This won't honor multi-release jars in this PR.

Reply:
Yes, but hopefully the MultiReleaseJarFile PR will take care of that with minimal changes. Or maybe this PR should wait for the MultiReleaseJarFile PR to get merged?